packages/react/src/hooks/useAuth.ts

```diff
@@ -127,20 +143,11 @@ export const useAuth = (initialAuthState: any = {}): UseAuthReturn => {
  * } = useDerivedAuth(authObject);
  * ```
  */
-export function useDerivedAuth(authObject: any): UseAuthReturn {
-  const {
-    sessionId,
-    userId,
-    actor,
-    orgId,
-    orgSlug,
-    orgRole,
-    has,
-    signOut,
-    getToken,
-    orgPermissions,
-    factorVerificationAge,
-  } = authObject ?? {};
+export function useDerivedAuth(authObject: any, options: PendingSessionOptions = {}): UseAuthReturn {
+  const optionsContext = useOptionsContext();
```

Comment:
Starting to get this error: 

> <w> [webpack.cache.PackFileCacheStrategy] Serializing big strings (105kiB) impacts deserialization performance (consider using Buffer instead and decode when needed)

I think this is due to `useOptionsContext` being added here, currently investigating